tests/inventory/host_vars/single_custom_kayobe_argument.yml

```diff
@@ -0,0 +1,9 @@
+---
+github_output_directory: .github/single_custom_kayobe_argument
+
+github_environment_selector: single
+
+github_kayobe_environments: [prod-gb-01]
```

Comment:
Does this environment actually end up in the generated file? AFAICT it will still use production. Would it be better to remove KAYOBE_ENVIRONMENT from the default args and take it from this list when the selector is `single`?

Reply:
Taken from the most recent set of artefacts.

```yaml
     run: |
       /src/.automation/pipeline/overcloud-host-package-update.sh
     env:
       KAYOBE_ENVIRONMENT: 'prod-gb-01'
```